sw/airborne/modules/datalink/telemetry.c

```diff
@@ -112,7 +112,7 @@ void telemetry_reporting_task(void)
  * @param _mode telemetry mode
  * @param _id id of the message in telemetry system (see var/<AC>/generated/periodic_telemetry.h)
  */
-void periodic_telemetry_err_report(uint8_t _process, uint8_t _mode, uint8_t _id)
+void periodic_telemetry_err_report(uint8_t _process, uint8_t _mode, uint16_t _id)
```

Comment:
note that this parameter is cast to uint8 to be sent with the `TELEMETRY_ERR` message

Reply:
I know, didn't want to change the message. But it might be better to do that